test/index.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var expect = require('expect');
+
+var logger = require('../');
+
+describe('gulplog', function() {
+
+  afterEach(function(done) {
```

Comment:
```suggestion
  after(function(done) {
```
The logger only gets configured once, so if you remove afterEach it'll fail if we add another test.